modules/ROOT/pages/api-groups-landing-page.adoc

```diff
@@ -0,0 +1,69 @@
+= API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+API Groups enables you to bundle your APIs and specific resources to solve specific user needs. Instead of individually using a list of resources and APIs, the API users can request access to and use these APIs as a package that solves a specific problem for them. 
+
+When you create an API Group, you can create different instances of that group with different SLAs and rate limit conditions based on the problem it must solve. You can then publish the API Group to Anypoint Exchange for users to be able to subscribe to use the package.
+
+Because each API Group can include several API instances, and each instance can be included in multiple API Group solutions, package customization options are limitless. 
+
+== How API Groups Works
+
+As an API Group Administrator, you create an API Group for a specific organization and environment. API Groups are versioned, like the APIs, to easily maintain changes.  
+
+You can create instances of an API Group in the same or different environments and add different SLA and rate limit conditions. 
+
+The following diagram explains how you can implement API Groups in different environments.
+image:api-groups-hierarchy.png[]
+
+
+== Use Case
+
+To better understand how API Groups facilitates API consumer engagement, let us look at the example of the Good Weather company. Good Weather is a weather forecasting company that  predicts the atmospheric conditions for a given location and time. 
+
+Good Weather collects quantitative data about the current state of the atmosphere at a given place and using meteorology. Good Weather enables consumers to access their weather forecast data using the following APIs:
+
+* Locations API
+* Forecast API
+* Current Conditions API
+* Indices API
+* Weather Alarms API
+* Alerts API
+* Imagery API
+* Tropical API
+* Translations API
+
+Currently, the consumers access and use the APIs by:
+
+* Scanning the Good Weather company portal to find appropriate APIs to actualize a business flow. The consumers have to determine the API they need by reviewing a long list of APIs on the portal. Additionally, the problem is compounded when developers modify the existing APIs or introduce additional APIs. 
```

Comment:
GLOBALLY fix each bullet item to use this format:
```suggestion
* Searching the Good Weather company portal to find appropriate APIs to actualize a business flow +
To determine the precise API they need, customers have to search a long list of APIs on the portal. This difficulty is compounded when developers modify the existing APIs or introduce additional APIs. 
```

Reply:
Done! Thanks!